client/src/components/TypeList.vue

```diff
@@ -237,6 +237,11 @@ export default Vue.extend({
 </template>
 
 <style scoped lang='scss'>
+.type-checkbox {
+  max-width: 80%;
+  overflow: hidden;
```

Comment:
```suggestion
  overflow-wrap: anywhere;
```
My original  `overflow: hidden` was probably the wrong thing to do.
This doesn't maintain item height for all types, but will prevent the name from being cut off.  Other option is to truncate it with a hover tooltip of the full name.

![image](https://user-images.githubusercontent.com/61746913/86472724-a04f5680-bd0d-11ea-937a-7d5eed0c1073.png)